app/templates/components/global-footer/global-footer-bottom-bar.hbs

```diff
@@ -13,9 +13,10 @@
 			ns='design-system'
 		}}}
 	</div>
-	<div class="wds-global-footer__bottom-bar-row wds-has-border-top global-footer-full-site-link-wrapper">
```

Comment:
If we'd leave the "global-footer-full-site-link-wrapper" on a wrapper, then we could have a much more specific selector in acceptance tests. What do you think?

Also, did you remove it because it wasn't used anywhere?

Reply:
Yeah, it wasn't used anywhere else.